### ISSUE TITLE
fix wall/region with style lj1043 not working

### ISSUE THREAD

**Issue by dailyllama** [lammps/lammps#795]
## Summary

Running a fix wall/region as described [here](http://lammps.sandia.gov/doc/fix_wall_region.html)

The wall/region is a sphere with atoms enclosed. Applying fix wall/region works with styles lj126 and lj93, but not with lj1043.

## Type of Issue

```
ERROR: Illegal fix wall/region command (../fix_wall_region.cpp:64)
Last command: fix             1 all wall/region 1 lj1043 1.5e-12 3.0 25.0
```

## Detailed Description (Enhancement Suggestion)

I expected this potential to work as with the other potentials. I checked the source file fix_wall_region.cpp, but I do not see an obvious reason for this bug and am not proficient enough to fix it myself yet.

## LAMMPS Version (Bug Report)

Lammps versions 11 Aug 2017 and 23 Oct 2017 both have the same error.

## Steps to Reproduce (Bug Report)

Can run an input file such as:

```
units		metal	
dimension	3
atom_style	molecular        
boundary        f f f 		
read_data       ${infile}
group           1 type 1
region 		1 sphere 0.0 0.0 0.0 15.0	
variable        dt equal 0.0005   
pair_style	lj/cut 25.0	     
pair_modify     tail yes
pair_coeff      1 1 1.0e-12 3.4

# These do work
#fix             1 all wall/region 1 lj126 1.5e-12 3.0 25.0
#fix             1 all wall/region 1 lj93 1.5e-12 3.0 25.0

# This does not work
fix             1 all wall/region 1 lj1043 1.5e-12 3.0 25.0

velocity	all create 100.0 12345 dist uniform
reset_timestep  0           
fix		2 all nvt temp 100.0 100.0 0.05
thermo          500
thermo_style 	custom step pe press temp
run		5000
```

An infile to put some atoms in the sphere can be read through read_data. Example:

```
LAMMPS Data File

5 atoms

1 atom types

-50.0 50.0 xlo xhi
-50.0 50.0 ylo yhi
-50.0 50.0 zlo zhi

Masses

1 1.0

Atoms

1 1 1 2.29555509883 -0.952428175922 11.6895986752
2 2 1 1.15790640799 -11.9778064415 -0.00375804898562
3 3 1 4.96395423378 8.71875437665 6.45586091398
4 4 1 10.0335220896 0.768417494719 5.92772006945
5 5 1 -7.9243502079 -6.60171560064 5.91953433267

```

Then this is run with a command such as:

`mpirun -np 4 lmp_mpi -in INPUTSCRIPT -var infile READDATAFILE`

**Comment by Pakketeretet2** [comment]
Your LAMMPS seems out of date, as line 64 is now 'else if (strcmp(arg[4],"
lj126") == 0) style = LJ126;', the error couldn't have been emitted from
there. Possibly your version is outdated and does not support lj1043?

**Comment by Pakketeretet2** [comment]
I cannot reproduce this with versions Oct 23 2017 nor with Jan 17 2018. Please update your LAMMPS.

**Comment by akohlmey** [comment]
This is due to a bug that was fixed in commit e196a2b9e54ad207a72b15e501e6fa1f9b4893f1 in September 2017, i.e. version 23 October 2017 should already work.

**Comment by dailyllama** [comment]
Confirm that updating fixed the issue. Thanks.
